frontend/views/containers/proposals/ProposalVoteOptions.vue

```diff
@@ -107,7 +107,7 @@ export default ({
         if (oneVoteToPass(proposalHash)) {
           if (this.type === PROPOSAL_INVITE_MEMBER) {
             passPayload = await createInvite({
-              invitee: this.proposal.data.proposalData.memberName,
+              invitee: this.proposal.data.proposalData.memberID,
```

Comment:
Same isue here, I'm pretty sure `memberID` isn't the right name for the variable